### PR TITLE
Ltd 4126 appeal details

### PR DESCRIPTION
**Pull request by kevincarrogan**
### Aim

Adds an endpoint to get the details of an appeal document.

Adds appeal to the standard application serializer.

[LTD-4126](https://uktrade.atlassian.net/browse/LTD-4126)

### CODE REVIEW

File: api/appeals/factories.py [hunk 1]
```diff
@@ -1,8 +1,21 @@
 import factory
 
-from .models import Appeal
+from .models import (
```
Comment (saruniitr) [inline api/appeals/factories.py:3]: Can this file be moved to `tests` directory as it is only used in tests?

Comment (kevincarrogan) [reply api/appeals/factories.py]: Done.

File: api/appeals/tests/factories.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import factory
+
+from ..models import (
```
Comment (currycoder) [inline api/appeals/tests/factories.py:3]: Not to say that this is invalid, but generally we tend to use absolute import paths.  Do we want to keep that consistent?

**Review by currycoder — APPROVED** [review]

File: api/appeals/tests/factories.py (hunk 2 above)
Comment (kevincarrogan) [reply api/appeals/tests/factories.py:3]: We should use absolute imports when it's across Djangos apps but we should use relative imports when it's within a Django app is my personal feeling.

**Pull request merged**